homeassistant/components/homematicip_cloud/climate.py

```diff
@@ -96,3 +102,11 @@ def max_temp(self) -> float:
         if temperature is None:
             return
         await self._device.set_point_temperature(temperature)
+
+
+def _get_first_heating_thermostat(heating_group: AsyncHeatingGroup):
+    """Return the first HeatingThermostat from a HeatingGroup."""
+    for device in heating_group.devices:
+        if isinstance(device, (AsyncHeatingThermostat,
+                               AsyncHeatingThermostatCompact)):
+            return device
```

Comment:
We should always return something explicitly if we return something anywhere. `return None` in the other case.